irodsManager.py

```diff
@@ -9,9 +9,16 @@ def __init__(self, url_base: str):
         self.token = None
 
         self.collections = self.collections_manager(self.url_base)
+        self.data_objects = self.data_objects_manager(self.url_base)
+        self.information = self.information_manager(self.url_base)
+        self.queries = self.query_manager(self.url_base)
+        self.resources = self.resources_manager(self.url_base)
+        self.rules = self.rules_manager(self.url_base)
+        self.tickets = self.ticket_manager(self.url_base)
+        self.users = self.user_manager(self.url_base)
+        self.zones = self.zone_manager(self.url_base)

```

Comment:
Consider moving the logic for each endpoint into its own module. That will reduce the number of things you (and the next developer) have to track when editing the code.

Reply:
Opened issue #16